Formula/d/displayplacer.rb

```diff
@@ -0,0 +1,21 @@
+class Displayplacer < Formula
+  desc "Utility to configure multi-display resolutions and arrangements"
+  homepage "https://github.com/jakehilborn/displayplacer"
+
+  url "https://github.com/jakehilborn/displayplacer/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "54b239359dbf9dc9b3a25e41a372eafb1de6c3131fe7fed37da53da77189b600"
+  license "MIT"
+
+  head "https://github.com/jakehilborn/displayplacer.git", branch: "master"
+
+  depends_on :macos
+
+  def install
+    system "make", "-C", "src"
+    bin.install "src/displayplacer"
+  end
+
+  test do
+    system "#{bin}/displayplacer", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.